plutus-scb-client/.gitignore

```diff
@@ -1,6 +1,9 @@
 /node_modules/
```

Comment:
Kill this one too?

Reply:
Ah, I thought I did. Well spotted, thanks.